### PR TITLE
document the task to deploy lvmo

### PR DESCRIPTION
**Pull request by alknopfler**
Signed-off-by: Alberto Morgante Medina <alknopfler@users.noreply.github.com>

# Description

update doc for lvmo in order to create pv dynamically

Fixes # NO-ISSUE

## Type of change

Please select the appropriate options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] This change is a documentation update

## Testing

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Versions:
- Hardware:

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not be merged unless it passes CI or there is a comment/update saying what testing was passed.
- [ ] PRs should not be merged unless positively reviewed.

### CODE REVIEW

File: documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc [hunk 1]
```diff
@@ -38,6 +38,19 @@ Installer-provisioned installation of {product-title} requires:
...
+- In case you want to use dynamic PV creation using LVMO we've prepared a Tekton task to do that:
+```
+run-hub-lvmo-task:
+	tkn task start -n edgecluster-deployer \
+    			-p ztp-container-image="$(PIPE_IMAGE):$(BRANCH)" \
+    			-p edgeclusters-config="$$(cat $(EDGECLUSTERS_FILE))" \
+    			-p kubeconfig=${KUBECONFIG} \
+			    -w name=ztp,emptyDir="" \
+    			--timeout 5h \
+    			--pod-template ./pipelines/resources/common/pod-template.yaml \
```
Comment (achuzhoy) [inline documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc:50]: redundant for doc?

Comment (achuzhoy) [reply documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc:50]: I meant line 50 only as redundant :)

Comment (alknopfler) [reply documentation/ztp-for-factories/ztp-for-factory-prerequisites.adoc]: done!

**Review by flaper87 — APPROVED** [review]
LGTM

**Pull request merged**